qutebrowser/completion/completionwidget.py

```diff
@@ -93,15 +95,16 @@ class CompletionView(QTreeView):
     def __init__(self, win_id, parent=None):
         super().__init__(parent)
         self._win_id = win_id
+        self._completion_connected = False
```

Comment:
This is initially `False`, but the signals *are* connected in `statusbar/command.py` if `auto-open` is true - wouldn't that double-connect the signals?